app/models/tasks/assign_hearing_disposition_task.rb

```diff
@@ -126,6 +126,7 @@ def hold!
   def update_children_status_after_closed
     update_args = { status: status }
     update_args[:closed_at] = Time.zone.now if self.class.closed_statuses.include?(status)
```

Comment:
Want to clean this up while we're here?
```suggestion
    update_args[:closed_at] = Time.zone.now unless open?
```